.githooks/pre-push/beachball-check.js

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env node
+process.argv.push('check');
```

Comment:
Oh this calls "beachball check" but uses whatever is installed instead of shelling out another process I guess... This adds a pre-push beachball check call to make sure that we are sane with change files before we even submit for a build

Reply:
Yes, I added some comments there!